tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -72,6 +73,8 @@ protected TaskExecutor() throws IOException {
     this.tbPort = this.tbSocket.getLocalPort();
     this.gatewayServerSocket = new ServerSocket(0);
     this.gatewayServerPort = this.gatewayServerSocket.getLocalPort();
+    framework = TonyConfigurationKeys.MLFramework.valueOf(tonyConf.get(TonyConfigurationKeys.FRAMEWORK_NAME,
```

Comment:
Can we be consistent and either use `this.` everywhere or not use `this.` everywhere? My personal preference is to omit `this.` and name member variables with a `_` prefix.